download-libs.sh

```diff
@@ -38,39 +38,37 @@ fi
 # checkout closure compiler
 if [ ! -d closure-compiler/.git ]; then
   if [ -d closure-compiler ]; then # remove binary release directory
-    rm -rf closure-compiler
+    rm -rf closure-compiler 
   fi
-  git clone --depth 1 https://github.com/google/closure-compiler/ closure-compiler
-fi
-
-# build closure compiler
-if [ ! -f closure-compiler/build/compiler.jar ]; then
+  git clone https://github.com/google/closure-compiler
   cd closure-compiler
   ant jar
   cd ..
 fi
 
 # checkout closure templates compiler
-if [ ! -d closure-templates-compiler ]; then
-  curl https://dl.google.com/closure-templates/closure-templates-for-javascript-latest.zip -O # -k --ssl-added-and-removed-here-;-)
-  unzip closure-templates-for-javascript-latest.zip -d closure-templates-compiler
-  rm closure-templates-for-javascript-latest.zip
+if [ ! -d closure-templates ]; then
+  git clone https://github.com/google/closure-templates
+  cd closure-templates
+  # hack because things there's a warning on OSX if you're using Java 8
```

Comment:
Please add an if..then to narrow it down to Java 1.8, errors are errors for a reason ;)